### DISCUSSION TITLE
Username change doesnt update API with new username

### DISCUSSION THREAD

**Discussion by whyvn** [community/community#38456] [Mobile]
I changed my username recently and things like the starred repos, discussions and lists were all broken since it was requesting from my old username which doesnt exist anymore and this only happens on the mobile. To reproduce just change your username while logged in on he mobile app. And to fix this i just had to logout.

**Comment by radar-blip** [comment]
Changing your username can have[ unintended side effects](https://docs.github.com/articles/what-happens-when-i-change-my-username/).

**Reply by whyvn** [reply]
Ah thanks but is this an issue thats being fixed?

**Reply by radar-blip** [reply]
I dont think there is a fix.
It's the effects of changing your username since your username is connected to to your account and everything related to your account.

**Reply by whyvn** [reply]
oh yeah i don't think ther is anything i could do about it but im just wondering if its a bug that's being worked on.

**Reply by radar-blip** [reply]
Nope, not a bug.

**Reply by aconsuegra** [reply]
Logging out and back in to the app should fix all the issues related to username changes.

**Reply by whyvn** [reply]
Yeah i can easily fix it but how is this not a bug?
